src/Java.Interop/Java.Interop/JniRuntime.cs

```diff
@@ -277,8 +277,7 @@ public virtual string GetCurrentManagedThreadStackTrace (int skipFrames = 0, boo
 
 		public virtual void FailFast (string message)
 		{
-			var t = typeof (Environment).GetTypeInfo ();
-			var m = t.DeclaredMethods.FirstOrDefault (x => x.Name == "FailFast");
+			var m = typeof (Environment).GetMethod ("FailFast");
```

Comment:
This method was `Public` so default `BindingFlags` should work: https://github.com/mono/mono/blob/3563bd90dc54f74a33ca468d0e2eb0d189a89dc1/mcs/class/corlib/System/Environment.cs#L921

It looks like `Environment.FailFast` is still not in netstandard 2.0, so we have to use reflection here.